package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "nvs",
-	"version": "1.3.0",
+	"version": "1.3.1",
```

Comment:
I suppose in a project that is often cherry-picking changes across multiple major-version branches/releases, then you wouldn't want the changes to include version numbers. But this project isn't that complex, so it doesn't matter to me.

As for semver-minor vs patch, I think it could go either way. So, this is fine.